instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PostHeader from './PostHeader';
+import PostFooter from './PostFooter';
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+
+const PostWrapper = styled.div `
+    max-width: 600px;
+    margin: 20px auto;
+    border: 1px solid lightgrey;
+    background: white;
+    @media (max-width: 650px) {
```

Comment:
great use of media queries, try looking into theming so that you can maybe just import it and don't have to remember what the mobile with should be.